package/MDAnalysis/coordinates/XDR.py

```diff
@@ -43,6 +43,22 @@ def offsets_filename(filename, ending='npz'):
                                                         ending=ending))
 
 
+def read_numpy_offsets(filename):
```

Comment:
With offset generalization in mind, as @richardjgowers has mentioned, would it make sense to put this under `lib`?